src/service/agent.js

```diff
@@ -177,6 +179,18 @@ async function executeJob(jobInfo, keepFiles) {
     downloader.logger = jLogger;
     await downloader.download(tmpDirPath);
 
+    // if the extraFiles is not provided, the agent will work as normal flow
```

Comment:
move this logic to KatalonCommandExecutor.preExecuteHook

Reply:
Ok a